controls/V-72059.rb

```diff
@@ -58,11 +58,11 @@
 system/partition."
   tag "fix_id": "F-78411r1_fix"
 
-  IGNORE_SHELLS = NON_INTERACTIVE_SHELLS.join('|')
+  ignore_shells = non_interactive_shells.join('|')
 
   # excluding root because its home directory is usually "/root" (mountpoint "/")
-  users.where{ !shell.match(IGNORE_SHELLS) && (uid >= 1000)}.entries.each do |user_info|
-    next if EXEMPT_HOME_USERS.include?("#{user_info.username}")
+  users.where{ !shell.match(ignore_shells) && (uid >= 1000)}.entries.each do |user_info|
```

Comment:
This should be `UID_MIN` from the target system instead of `1000` since that can be arbitrarily set by the user.